docs/notebooks/demo_Cometary_Activity.ipynb

```diff
@@ -0,0 +1,1047 @@
DotfBafsS3O++eYbIiIiqFy5Mr179862/VoLq0wIAwMDWbBggcmy/fv3GweI8fX1pVOnThQoUID33nuPQ4cOsW7dOqKionBxcaF69eq89dZbDBo0CFvbjH8z0Lx5cwIDAxk3bhwrVqwgMTGRqlWr8uWXX/Lmm29m6zkKIYQQQgghntGJ2/DGWn0y+FJx+ONVcMm+wVtu3brFjBkzAJg8eTJ2dlaZPj0TqzyjgIAAAgICnlrO3d2dmTNnZmrfvr6+KIqS5vr69euzadOmTO1TCCGEEEIIkcv+vQXd10FMIrxYDJa+Cq4O2XqIr776iri4OF588UU6duyYrfu2FlaZEAohhBBCCCFEmg6EQa/18EAH9YvBsg7ZngwmJydz7NgxAKZOnZrl6SasnSSEQgghhBBCiOfHnmvQewM8TIImJWFh+2xPBgFsbW0JDAxkz549NGvWLNv3by0kIRRCCCGEEEI8H7YGw4DNkJAMLX1g/svZOoDMk2xsbGjevHmO7d8aPNfTTgghhBBCCCHyiVUXoO8mfTLYviwsaJ8jyWBKSgo//fQT9+/fz/Z9WyNJCIUQQgghhBDW7beTMHQbJKXAGxVgTlvQZs88g09avHgxw4YNo27duiQnJ+fIMayJNBkVQgghhBBCWCdFgWmH4dsj+seDa8BXTcAmZwZ4iY2NZfTo0QAMGDAgU1PYPa8kIRRCCCGEEEJYn6QU+HQPLDijf/zpizCiLuTgaJ/Tp0/nxo0blClThg8++CDHjmNNJCEUQgghhBBCWJfYRBi0BXaEggaY5g/9q+foIUNDQ/n6668B+Oabb3B0dMzR41kLSQiFEEIIIYQQ1uNmLPRcD2fC9YPG/NpGP4hMDhs1ahTx8fH4+/vTuXPnHD+etZCEUAghhBBCCGEdTodDr3Vw8wEUcoI/XoVaRXL8sJs3b2bVqlXY2tryww8/5NlJ6C2RhFAIIYQQQgihvu2hMGgzPNBBBS9Y2gFKu+fKoWvUqEH37t0pXrw4NWvWzJVjWgtJCIUQQgghhBDqCjitH0AmWYEmJfUTzntoc+3wxYsXZ9myZaSkpOTaMa2FJIRCCCGEEEIIdaQo8OUBmHlc/7hHJfi2OTjkznQP8fHxJoPH2Njkv2na898ZCyGEEEIIIdR3PxH6bnycDH76IvzYMteSQUVR6NChA926dePGjRu5ckxrJAmhEEIIIYQQInddioQ2K2BzMGhtYVZrGFkvR+cYfNLy5cvZvn07f//9Nw8fPsy141obaTIqhBBCCCGEyD2bg2HoVojVQXFXCHg5V0YSTS08PJxhw4YBMGbMGMqVK5erx7cmkhAKIYQQQgghcl6KAt8egemH9Y9fKg5z20Fh51wP5YMPPuDu3btUq1aNTz/9NNePb00kIRRCCCGEEELkrPuJ8N422BSsfzywOnzZGOxzp79gauvXr2fJkiXY2Ngwb948HBwccj0GayIJoRBCCCGEECLnXI6EPhv1/Qa1tvB1M+hZWZVQoqOjeeeddwAYMWIE9erVUyUOayIJoRBCCCGEECJn/H0ZPtypryEs5gIL2ud6f8HUQkJCsLGxoXz58kyYMEG1OKyJJIRCCCGEEEKI7BWfBGP3w/xT+scq9hdMrWbNmpw5c4br16/j7KxuLNZCEkIhhBBCCCFE9rkSBYO2wKm7+sfDauvnGFShv6Albm5uVK6sTpNVayTzEAohhBBCCCGyx1+XoOVyfTJY0BGWdoAvGqqeDL777rv8+uuvKIqiahzWSGoIhRBCCCGEEM/mYRJ8sQ8WnNE/fqk4/NYGirmqGxewevVqZs2ahY2NDY0aNaJ69epqh2RVJCEUQgghhBBCZF1QJAzcAmfCQQN8WBc+rg926jdGvHXrFm+//TYAn3zyiSSDFkhCKIQQQgghhMg8RYHFZ+HzQIjTgbcTzGoNzUqrHRkAiqIwePBg7t27R82aNRk/frzaIVklSQiFEEIIIYQQmXM3Dkbsgs2PJppvXEKfDBZVv4mowezZs1m/fj0ODg4sWrQo309AnxZJCIUQQgghhBAZtzUEPtwBdx+Cgw181gDeeQFsNGpHZnT69GmGDx8OwOTJk6WpaDokIRRCCCGEEEI83QMdjN8PAaf1jysVgFltoJq3unFZcOjQIRITE2nXrp0xMRSWSUIohBBCCCGESN/x2zB0GwRF6R+/U1NfM+honenEoEGDqFSpEn5+ftjYqD+4jTWzzjsohBBCCCGEUJ8uGb7/F749AskKFHOBma2gaSm1I3uqxo0bqx3Cc0HSZSGEEEIIIYS5/+5C65Uw/bA+GexYHvb0tNpkMDQ0lBYtWnDu3Dm1Q3muSEIohBBCCCGEeCwhGaYcgjYr9HMLFnDUTzL/e1vwclQ7OosSEhLo2rUru3bt4t1331U7nOeKNBkVQgghhBBC6J24De/vgPMR+sevlYepTaGQs7pxPcWoUaM4cuQIXl5eBAQEqB3Oc0USQiGEEEIIIfK7+CT4+jD8fFzfPNTbCab56xNCK7d8+XJmzpwJwKJFi/Dx8VE5oueLJIRCCCGEEELkZ4dvwvCdcDFS/7iLH0xuCgWd1I0rA86fP8+gQYMAGDNmDK+88orKET1/JCEUQgghhBAiP4qKh68OwoIz+seFneGbZvByWVXDyqjY2FjeeOMNYmNjad68ORMmTFA7pOeSJIRCCCGEEELkJ4oCf12CzwPhbpx+2ZtVYFxDqx00xpKHDx/i5eVFsWLFWLJkCXZ2ktpkhVw1IYQQQggh8ovQGBizH3Zd1T/289LXCjYsoWpYWVGoUCF27NhBSEgIRYsWVTuc55YkhEIIIYQQQuR1umT8NsZg978VEJ8MWlsYXhf+V1v/93Pk1q1bxgTQwcGBChUqqBzR803mIRRCCCGEECIv2x+GXZs/qbI6Gk18MjQpCXt6wMh6z10yeOnSJSpXrszIkSNJSkpSO5w8QWoIhRBCCCGEyItuxsL4/bD6EhogwdUG20n+2PWsAhqN2tFlWnR0NJ06
```

Comment:
nucelus --> nucleus

"observed to [be] much brighter"

Reply:
thanks!